serverless/cmd/feeder/impl/feeder.go

```diff
@@ -42,7 +42,7 @@ type Witness interface {
 	// Must return os.ErrNotExists if the logID is known, but it has no checkpoint for that log.
 	GetLatestCheckpoint(ctx context.Context, logID string) ([]byte, error)
 	// Update attempts to clock the witness forward for the given logID.
-	Update(ctx context.Context, logID string, newCP []byte, proof [][]byte) error
+	Update(ctx context.Context, logID string, newCP []byte, proof [][]byte) ([]byte, error)
```

Comment:
Nit: maybe the comment could make it a bit clearer what is being returned here?

Reply:
Good call, done.